cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -68,3 +73,16 @@ func ValidateTokenDiscovery(c *kubeadm.TokenDiscovery, fldPath *field.Path) fiel
 	allErrs := field.ErrorList{}
 	return allErrs
 }
+
+func ValidateServiceSubnet(subnet string, fldPath *field.Path) field.ErrorList {
```

Comment:
I really think tests should be added as new code is added.  I think it is one of the things we agreed on as a SIG to improve the overall quality of kubeadm.